CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 * **[Fix]** Disable module debugging for release mode in the SDK to fix dSYM warnings.
 
+### App Center Distribute
+
+* **[Fix]** Fix a crash in some case when `SFAuthenticationSession` accesses the controller which is in the process of being released.
+* **[Fix]** Fix sign-in with authenticator app during updates activation.
```

Comment:
Can you revise this to be more detailed and generic? Is it only caused by Authenticator app or it could be possible to be crashed with others?

Reply:
Updated